client/src/pages/HomePage/HomePage.js

```diff
@@ -1,5 +1,107 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
+import { Button, Container, Typography } from '@mui/material';
+import ArrowRightAltIcon from '@mui/icons-material/ArrowRightAlt';
+import { styled } from '@mui/material/styles';
```

Comment:
You can combine "styled" with other components from "@mui/material"

Reply:
please check the below screenshot from the official documentation.

![image](https://user-images.githubusercontent.com/107735322/198905886-16c32d73-372d-49e5-a5b4-a6472cc4d8ec.png)

Reply:
Yes, but you also can use `import { styled, ... } from '@mui/material'`, as you already using this path